### PR TITLE
feat(install): display filtered explorer link

### PR DESCRIPTION
**Pull request by ctrombley**
### Testing
`./bin/darwin/newrelic installTest -s DISPLAY_EXPLORER_LINK -n test-display-explorer-link`

An end to end test against the java APM recipe can also be used.

### CODE REVIEW

File: internal/install/execution/success_link_generator.go [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+import (
+	"fmt"
+	"strings"
+
+	"github.com/newrelic/newrelic-cli/internal/credentials"
+	"github.com/newrelic/newrelic-cli/internal/utils"
+	"github.com/newrelic/newrelic-client-go/pkg/region"
+)
+
+type SuccessLinkGenerator interface {
```
Comment (sanderblue) [inline internal/install/execution/success_link_generator.go:12]: Nice

**Review by zlesnr — APPROVED** [review]

File: internal/install/execution/success_link_generator.go [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+func (g *ConcreteSuccessLinkGenerator) GenerateExplorerLink(filter string) string {
+	return fmt.Sprintf("https://%s/launcher/nr1-core.explorer?platform[filters]=%s", getPlatformHost(), utils.Base64Encode(filter))
+}
+
+func (g *ConcreteSuccessLinkGenerator) GenerateEntityLink(entityGUID string) string {
+	return fmt.Sprintf("https://one.newrelic.com/redirect/entity/%s", entityGUID)
+}
+
+// getPlatformHost returns the host for the platform based oin the region set.
+func getPlatformHost() string {
```
Comment (sanderblue) [inline internal/install/execution/success_link_generator.go:32]: Nitpick and not a blocker either, but my first thought on this was OS/Platform such as CentOS, etc. Maybe `getNewRelicURLHostname()`

**Review by sanderblue — DISMISSED** [review]

File: internal/install/execution/success_link_generator.go (hunk 2 above)
Comment (ctrombley) [reply internal/install/execution/success_link_generator.go:32]: 👍

File: internal/install/execution/install_status.go [hunk 3]
```diff
@@ -212,14 +213,29 @@ func (s *InstallStatus) isCanceled() bool {
...
+func (s *InstallStatus) SetTargetedInstall() {
 	s.targetedInstall = true
 }
 
-func (s *InstallStatus) IsTargetdInstall() bool {
+func (s *InstallStatus) IsTargetedInstall() bool {
 	return s.targetedInstall
 }
 
+func (s *InstallStatus) RollupEntityGUID() string {
```
[the remark this answers is not in the document]
Comment (ctrombley) [reply internal/install/execution/install_status.go:224]: 👍

**Review by sanderblue — APPROVED** [review]

**Review by sanderblue — APPROVED** [review]

**Pull request merged**